srcpkgs/intel-parallel-studio-xe/template

```diff
@@ -0,0 +1,591 @@
+# Template file for 'intel-parallel-studio-xe'
+_remove_docs=true
+_remove_static_objects_mkl=false
+_remove_static_objects_ipp=false
+
+_year=2019
+_v_a=5
+_v_b=281
+_update=5
+_sp='cluster_edition'
+_icc_ver='19.0.5'
+_vtune_ver='2019.6.0.602217'
+_inspector_ver='2019.5.0.602103'
+_advisor_ver='2019.5.0.602216'
+_dir_nr=15809
+_parallel_studio_xe_dir="parallel_studio_xe${_year:+_${_year}}${_update:+_update${_update}}${_sp:+_${_sp}}"
```

Comment:
Why wouldn't _year and _sp be set?

Reply:
The last one might not be, depending on which edition one wants. However, I have professional license and it works with enterprise installer... So it seems a run-time detection is used, not a package software installer difference.